x/zoneconcierge/keeper/ibc_packet_btc_timestamp.go

```diff
@@ -238,33 +248,36 @@ func (k Keeper) BroadcastBTCTimestamps(
 			return err
 		}
 
-		btcTimestamp, err := k.createBTCTimestamp(ctx, consumerID, channel, finalizedInfo)
-		if err != nil {
-			k.Logger(sdkCtx).Error("failed to create BTC timestamp for consumer, skipping",
-				"channel", channel.ChannelId,
-				"consumerID", consumerID,
-				"error", err.Error(),
-			)
-			continue
-		}
-
-		packet := types.NewBTCTimestampPacketData(btcTimestamp)
-		if err := k.SendIBCPacket(ctx, channel, packet); err != nil {
-			if errors.Is(err, clienttypes.ErrClientNotActive) {
-				k.Logger(sdkCtx).Info("IBC client is not active, skipping consumer",
+		// Send to all channels for this consumer
+		for _, channel := range channels {
```

Comment:
In my mind we are always only going to have one channel per consumer 

Any opinions here? @gusin13 @SebastianElvis @gitferry

Reply:
yep, there is always only one channel per consumer, see : https://github.com/babylonlabs-io/babylon/blob/v3.0.0-snapshot.250807/x/zoneconcierge/keeper/ibc_packet_btc_staking_consumer_event.go#L101
in which we enforce this during channel handshake